Intersect.Client.Framework/Input/ControlBinding.cs

```diff
@@ -34,7 +35,7 @@ public bool IsDown()
 
         if (IsMouseKey && Key.TryGetMouseButton(out var mouseButton))
         {
-            return gameInput.IsMouseButtonDown(mouseButton);
+            return gameInput.IsMouseButtonDown(mouseButton) && !DragAndDrop.IsDragging;
```

Comment:
This also needs to be in `WasDown` to avoid unexpected mouse "release" behavior.